concepts/tuples/about.md

```diff
@@ -1,4 +1,4 @@
-Tuples are used commonly to group information informally. A common pattern through Elixir is to group function return values with a status.
+Tuples are used commonly to group information informally. A common pattern in Elixir is to group function return values with a status.
```

Comment:
I wasn't sure if this was a grammar error or not... my hunch tells me it should be either "throughout" or "in".